packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component.ts

```diff
@@ -200,9 +201,20 @@ export class NgxFlickingComponent extends FlickingInterface
     const flicking = this._vanillaFlicking;
     if (!flicking) return;
 
-    this._ngZone.runOutsideAngular(() =>
-      flicking.renderer.forceRenderAllPanels()
-    );
+    this._ngZone.runOutsideAngular(() => {
+      void flicking.renderer.forceRenderAllPanels();
+    });
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { virtual, ...newOptions } = this.options;
+
+    // Omit 'virtual', as it can't have any setter
+    for (const key in newOptions) {
```

Comment:
The prop diff result is in the first parameter, try this one.

https://angular.io/api/core/OnChanges#ngOnChanges

Reply:
Well, there's no meaning on using that as Flicking bounds all options to prop `options`
![image](https://user-images.githubusercontent.com/26213435/199199861-fa55e099-fafa-49ac-96f8-b6493b9271bd.png)

Have to check every props inside `options` in either way.